<?xml version="1.0" ?>
<launch>
    <!--- TODO SASL
    <arg name="user" default="user"/>
    <arg name="password" default="pw"/>
    -->

    <arg name="security_protocol" default="ssl"/>
    <arg name="host" default="<hostname>"/>
    <arg name="endpoint_identification" default="none"/>

    <arg name="kafka_topic" default="chatter"/>

    <node pkg="ros_kafka" type="os_cloud_consumer" name="os_cloud_consumer" output="screen">
        <!---
        <param name="user" value="$(arg user)"/>
        <param name="password" value="$(arg password)"/>
        -->
        <param name="host" value="$(arg host)"/>
        <param name="kafka_topic" value="$(arg os_topic)"/>
        <param name="security_protocol" value="$(arg security_protocol)"/>
        <param name="endpoint_identification" value="$(arg endpoint_identification)"/>
        <param name="ca_cert" value="$(find ros_kafka)/certs/ca-cert"/>
        <param name="client_cert" value="$(find ros_kafka)/certs/client.pem"/>
        <param name="client_key" value="$(find ros_kafka)/certs/client.key"/>
        <param name="key_pw_var" value="CLIENT_PW"/>
        <!--- SPECIFIC FOR CONSUMER --> 
        <param name="group_id" value="test-group"/>
    </node>

</launch>

